frontend/src/components/Modal.js

```diff
@@ -0,0 +1,29 @@
+function Modal({ status }) {
+  return (
+    <div className="modal fade" id="myModal">
```

Comment:
Az id-t kiszedném, ha nem muszáj, mert ha valahol két Modal komponens is előfordul, akkor ütközik.